jobs/haproxy/spec

```diff
@@ -22,6 +22,9 @@ properties:
   ha_proxy.enable_proxy_protocol:
     description: "Enable port 81 for ProxyProtocol HTTP traffic. Will update X-Forwarded-Proto if dst_port was 443"
     default: ~
+  ha_proxy.enable_proxy_frontend:
+    description: "Enable port 8443 for HTTPS traffic."
+    default: ~
```

Comment:
⚠️ vexatious comment ⚠️ 

```suggestion
    default: false
```

`~` ends up as `nil` in ruby, and the following are equivalent:

```
if nil
  puts 1
end
```

```
if false
  puts 1
end
```

but default `false` communicates the intention better